sys/linux/auto.txt

```diff
@@ -3926,15 +3926,15 @@ ip_vs_dest_policy$auto_ip_vs_ctl [
 	IPVS_DEST_ATTR_STATS		nlattr[IPVS_DEST_ATTR_STATS, intptr]
 	IPVS_DEST_ATTR_ADDR_FAMILY	nlattr[IPVS_DEST_ATTR_ADDR_FAMILY, int16]
 	IPVS_DEST_ATTR_TUN_TYPE		nlattr[IPVS_DEST_ATTR_TUN_TYPE, int8]
-	IPVS_DEST_ATTR_TUN_PORT		nlattr[IPVS_DEST_ATTR_TUN_PORT, int16]
+	IPVS_DEST_ATTR_TUN_PORT		nlattr[IPVS_DEST_ATTR_TUN_PORT, sock_port]
 	IPVS_DEST_ATTR_TUN_FLAGS	nlattr[IPVS_DEST_ATTR_TUN_FLAGS, int16]
 ] [varlen]
 
 ip_vs_svc_policy$auto_ip_vs_ctl [
 	IPVS_SVC_ATTR_AF		nlattr[IPVS_SVC_ATTR_AF, int16]
 	IPVS_SVC_ATTR_PROTOCOL		nlattr[IPVS_SVC_ATTR_PROTOCOL, int16]
 	IPVS_SVC_ATTR_ADDR		nlattr[IPVS_SVC_ATTR_ADDR, int16]
```

Comment:
(Just a randomly checked attribute)

Why is it int16? It's NLA_BINARY
https://elixir.bootlin.com/linux/v6.11-rc6/source/net/netfilter/ipvs/ip_vs_ctl.c#L3213

Reply:
This is now fixed! NLA_BINARY was mistakenly translated into int16.